shopwareerpconnect/delivery_view.xml

```diff
@@ -8,7 +8,7 @@
             <field name="inherit_id" ref="delivery.view_delivery_carrier_form" />
             <field name="arch" type="xml">
                  <data>
-                    <field name="product_id" select="1" position="after">
+                    <field name="price_rule_ids" position="after">
                         <field name="shopware_code"/>
```

Comment:
v9 you had product_type, cannot we use it here?

Reply:
In inherited view of version 10 it does not have "product_type" field.